### ISSUE TITLE
JavaScript Error in IE8 due to Date.now();

### ISSUE THREAD

**Issue by Schleuse** [pyrsmk/qwest#86]
IE8 doesn't implement the *now* method on the *Date* object. I suggest using a local method to provide the current timestamp wich includes a polyfill:

```javascript
var getTime = Date.now || function () {
  return (new Date()).getTime();
};
```

Here are the occurences of Date.now():

https://github.com/pyrsmk/qwest/search?utf8=%E2%9C%93&q=Date.now

**Comment by pyrsmk** [comment]
Ah, right! I'll fix that ASAP

**Comment by pyrsmk** [comment]
Fixed with 2.2.1

**Comment by pyrsmk** [comment]
Oops, I forgot to publish to NPM. I'm fixing it
